clippy_tests/examples/range.stderr

```diff
@@ -1,11 +1,3 @@
-error: Range::step_by(0) produces an infinite iterator. Consider using `std::iter::repeat()` instead
```

Comment:
Since #95 no longer applies I'd vote for accepting the changeset and deprecating the lint later. 

cc @llogiq